contract_update_last_date_invoiced/wizards/update_last_date_invoiced.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2019 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class PrgEditLastDateInvoicedWizard(models.TransientModel):
+    _name = "contract.update.last.date.invoiced"
+    _description = "Update Contract Last Date Invoiced Wizard"
+
+    contract_line_id = fields.Many2one(
+        comodel_name="contract.line",
+        string="Contract Line",
+        required=True,
+        ondelete="cascade",
+    )
+    last_date_invoiced = fields.Date()
+    recurring_next_date = fields.Date()
```

Comment:
```suggestion
    recurring_next_date = fields.Date(string="Date of Next Invoice")
```
As @fcvalgar said, it's confusing that the fields have different labels.